relationships/synthesis/EXT-SERVICE-to-INFRA-REDSHIFTCLUSTER.yml

```diff
@@ -3,13 +3,18 @@ relationships:
     version: "1"
     origins:
       - Distributed Tracing
+      - OpenTelemetry
     conditions:
       - attribute: eventType
         anyOf: [ "Span" ]
-      - attribute: instrumentation.provider
-        anyOf: [ "opentelemetry" ]
+      - attribute: newrelic.source 
+        anyOf: [ "api.traces.otlp" ] 
+      - attribute: entity.type 
+        anyOf: [ "SERVICE" ]
       - attribute: rpc.service
         anyOf: [ "Redshift" ]
```

Comment:
What instrumentation are you using for Redshift that sets `rpc.service=Redshift`?